docker/jupyter/Dockerfile

```diff
@@ -289,9 +293,6 @@ RUN apt-get update \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-# fixed current ggplot issue where it imports Timestamp from pandas.lib (deprecated) instead of pandas
```

Comment:
Decided to embed this in the respective python layers instead of creating a new layer